### PR TITLE
Updated blockReassurance in composer.json

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | If we install the blockReassurance module from module, the version 3.x is installed
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? |  Relative to #14700
| How to test?  | If we install the blockReassurance module from module, the version 4.1.1 is installed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15501)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
Does this one need a specific version of blockreassurance module?

**Review by jolelievre — DISMISSED** [review]

**Comment by eternoendless** [comment]
IMO this PR should also include the new version of the module in composer

**Comment by Progi1984** [comment]
@eternoendless I do it :)

File: composer.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
         "ezyang/htmlpurifier": "^4.10",
         "geoip2/geoip2": "~2.4.2",
         "greenlion/php-sql-parser": "^4.1",
+        "guzzlehttp/cache-subscriber": "dev-master",
```
Comment (PierreRambaud) [inline composer.json]: What is the reason to add this dependency?

Comment (Progi1984) [reply composer.json]: Error :stuck_out_tongue:

**Review by PierreRambaud — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**